api/game-lobbies/game-lobbies.js

```diff
@@ -52,6 +52,13 @@ GameLobbies.schema = new SimpleSchema({
     index: 1
   },
 
+  endReason: {
+    label: "Ended Reason",
+    type: String,
+    optional: true,
+    regEx: /[a-zA-Z0-9_]+/
```

Comment:
This regexp does not seems to allow spaces, yet below I see `endReason: "finished early"`. Do I understand this wrong? :)

Reply:
yes, space shouldn't be allowed, sorry my bad.